mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -5245,6 +5245,33 @@ pub enum CancelBy {
     Coin { ticker: String },
 }
 
+pub async fn get_matching_orders(ctx: &MmArc, coins: &[String]) -> Result<Vec<Uuid>, String> {
+    let ordermatch_ctx = try_s!(OrdermatchContext::from_ctx(ctx));
+    let mut matching_orders = vec![];
+
+    let maker_orders = ordermatch_ctx.maker_orders_ctx.lock().orders.clone();
+    let taker_orders = ordermatch_ctx.my_taker_orders.lock().await;
+
+    for (uuid, order) in maker_orders.iter() {
+        let order = order.lock().await.clone();
+        if (coins.contains(&order.base.to_string()) || coins.contains(&order.rel.to_string()))
```

Comment:
It's not necessary to do `&order.base.to_string()`. It can be `&order.base`. Same for `rel`